tools/run_tests/run_interop_tests.py

```diff
@@ -288,22 +291,54 @@ def bash_login_cmdline(cmdline):
   return ['bash', '-l', '-c', ' '.join(cmdline)]
 
 
-def cloud_to_prod_jobspec(language, test_case, docker_image=None):
+def add_auth_options(language, test_case, cmdline, env):
+  """Returns (cmdline, env) tuple with cloud_to_prod_auth test options."""
+
+  language = str(language)
+  cmdline = list(cmdline)
+  env = env.copy()
+
+  # TODO(jtattermusch): this file path only works inside docker
+  key_filepath = '/root/service_account/stubbyCloudTestingTest-ee3fce360ac5.json'
```

Comment:
Minor: The key will be in the $HOME directory of the containers (as per what we did in build_interop.sh). So just in case, it might make sense to define a new variable - something like DOCKER_USER_HOME_DIR - or something like and make it "/root" instead of hardcoding /root/ in the key_filepath